openpype/settings/entities/schemas/projects_schema/schema_project_maya.json

```diff
@@ -53,6 +53,21 @@
                 }
             ]
         },
+        {
+            "type": "dict",
```

Comment:
I think this should be multiline text instead of list of strings
```
{
    "type": "text",
    "multiline": true,
    "key": "mel_workspace",
    "label": "Maya MEL Workspace",
}
```